sc-tools/sc-config-utils/sc-config/sc_config.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * This source file is part of an OSTIS project. For the latest info, see http://ostis.net
+ * Distributed under the MIT License
+ * (See accompanying file COPYING.MIT or copy at http://opensource.org/licenses/MIT)
+ */
+
+#include "sc_config.hpp"
+
+#include <string>
+#include <vector>
+
+extern "C"
+{
+#include "sc_config.h"
+}
+
+ScConfigGroup::ScConfigGroup(sc_config * config, std::string group)
+  : m_config(config)
+  , m_group(std::move(group))
+{
+  sc_list * keys = g_hash_table_get_keys(m_config);
```

Comment:
If we follow the "wrapper" policy then it's better not to use glib functions in wrapper code creating implicit dependencies. So I think we should create corresponding functionality on lower level in `sc_config.c`